### PR TITLE
Enable milestone and cherrypick plugins to k8s-sigs/Kueue

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch enables the following plugins to k8s-sigs/kueue
- milestone
- cherrypick

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -1625,6 +1630,12 @@ external_plugins:
       - issue_comment
       - pull_request
     endpoint: http://cherrypicker  
+  kubernetes-sigs/kueue:
+  - name: cherrypicker
```
Comment (alculquicondor) [inline config/prow/plugins.yaml:1634]: what does this do? :sweat_smile:

**Review by alculquicondor — COMMENTED** [review]
Don't we need to configure what milestones to add for each branch?

File: config/prow/plugins.yaml (hunk 1 above)
Comment (ArangoGutierrez) [reply config/prow/plugins.yaml:1634]: see example https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/190#issuecomment-1476294973

Comment (ArangoGutierrez) [reply config/prow/plugins.yaml:1634]: is a game changer for branch management

Comment (ArangoGutierrez) [reply config/prow/plugins.yaml:1634]: after the PR get's merged the bot creates a cherry-pick PR https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/192

Comment (alculquicondor) [reply config/prow/plugins.yaml:1637]: :exploding_head: 
great!

File: config/prow/plugins.yaml [hunk 2]
```diff
@@ -1313,6 +1316,11 @@ plugins:
     plugins:
     - override
 
+  kubernetes-sigs/kueue:
+    plugins:
+    - milestone
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply config/prow/plugins.yaml:1324]: now?

File: config/prow/plugins.yaml [hunk 3]
```diff
@@ -1313,6 +1319,11 @@ plugins:
...
+    - milestonestatus
```
Comment (alculquicondor) [inline config/prow/plugins.yaml:1325]: did you remove the cherrypicker?

what's the milestonestatus for?

Comment (ArangoGutierrez) [reply config/prow/plugins.yaml:1325]: cherrypicker is there, just enabled separately, see below

**Review by alculquicondor — COMMENTED** [review]
/lgtm

**Pull request merged**